plugin/src/main/scala/Giter8Plugin.scala

```diff
@@ -123,7 +123,9 @@ object Giter8Plugin extends sbt.AutoPlugin {
         } else Nil
 
         // copy test script or generate one
-        val script = new File(out, "test")
+        // the final script should always be called "test.script"
+        // no matter how it was originally called by user
+        val script = new File(out, finalScriptName)
```

Comment:
I originally thought this was a bug, but the final script is always called "test". That's what allows users to have scripts called "giter8.test" or "g8.test".

Unfortunately that conflicts with Play applications using default Play layout (see https://github.com/sbt/sbt/issues/630).

Since, sbt 1.2.1, scripted tests recognize "test.script" as a valid script name. We can workaround this situation by forcing it to be "test.script" when using sbt 1.x.